bazel/repository_locations.bzl

```diff
@@ -105,12 +105,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Abseil",
         project_desc = "Open source collection of C++ libraries drawn from the most fundamental pieces of Google’s internal codebase",
         project_url = "https://abseil.io/",
-        version = "5d8fc9192245f0ea67094af57399d7931d6bd53f",
-        sha256 = "e3812f256dd7347a33bf9d93a950cf356c61c0596842ff07d8154cd415145d83",
+        version = "20210324.1",
+        sha256 = "441db7c09a0565376ecacf0085b2d4c2bbedde6115d7773551bc116212c2a8d6",
```

Comment:
Latest is now https://github.com/abseil/abseil-cpp/releases/tag/20210324.2
```
version = "20210324.2",
sha256 = "59b862f50e710277f8ede96f083a5bb8d7c9595376146838b9580be90374ee1f",
release_date = "2021-06-01",
```

Reply:
done